src/api/imports.ts

```diff
@@ -74,7 +74,7 @@ function createImportsApi({
         if (data.type !== 'progress') break
       }
     },
-    importMBTiles(filePath: string, baseApiUrl: string) {
+    async importMBTiles(filePath: string, baseApiUrl: string) {
```

Comment:
Previously, this could resolve, reject, or synchronously throw. Now, it will always resolve or reject.

Relevant now because it's no longer "behind" and HTTP route.